pkg/router/template/template_helper.go

```diff
@@ -376,17 +376,33 @@ func parseIPList(list string) string {
 		return ""
 	}
 
+	var validIPs []string
+	var invalidIPs []string
+
 	ipList := strings.Fields(list)
 	for _, ip := range ipList {
-		if net.ParseIP(ip) == nil {
-			if _, _, err := net.ParseCIDR(ip); err != nil {
-				log.V(7).Info("parseIPList found not IP/CIDR item", "value", ip, "err", err)
-				return ""
-			}
+		// check if it's a valid IP
+		if net.ParseIP(ip) != nil {
+			validIPs = append(validIPs, ip)
+		} else if _, _, err := net.ParseCIDR(ip); err == nil {
+			// check if it's a valid CIDR
+			validIPs = append(validIPs, ip)
+		} else {
+			invalidIPs = append(invalidIPs, ip)
 		}
 	}
-	log.V(7).Info("parseIPList parsed the list", "value", list)
-	return list
+
+	if len(invalidIPs) > 0 {
+		log.V(7).Info("parseIPList found invalid IP/CIDR items", "invalidIPs", invalidIPs)
```

Comment:
I would keep the log line up at line 391 where you find an invalid IP, e.g. log it for each invalid ip. Then you don't need the invalidIPs slice, nor to take a len later.

Reply:
Thank you Candace!